include/github/apply.xml

```diff
@@ -125,6 +125,12 @@
                   <label>
                     <add>branch: bleeding</add>
                   </label>
+                  <comment src="issue/experimental.md">
+                    <tokens>
+                      <token name="version" value="8.0.0-BETA"/>
+                      <token name="stable" value="7"/>
```

Comment:
Be explicit: `7.0.0`.

Reply:
It may apply to 7.1 that will be released soon.

Reply:
`7.x` ?

Reply:
`7.x` works for me.